src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -209,11 +218,15 @@ private bool SignOut()
         /// <param name="isCheckboxChecked"></param>
         private void LaunchDynamo(bool isCheckboxChecked)
         {
-            viewModel.PreferenceSettings.EnableStaticSplashScreen = !isCheckboxChecked;
+            CloseWasExplicit = false;
+            if (viewModel != null)
```

Comment:
some null checks, as in my tests, I want to call these methods without the DynamoView actually being around.